From f17028feba97bcc2ce2ae7f2ee0dfb78075a571e Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 26 Feb 2019 16:52:46 +0100
Subject: [PATCH] power/opp: The variable tcpu_dev was renamed to cpu_np

The old variable is not used any more, remove it.
This fixes a compile warning.
---
 drivers/base/power/opp/of.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/base/power/opp/of.c b/drivers/base/power/opp/of.c
index 132799ad9e0a..93773655c82f 100644
--- a/drivers/base/power/opp/of.c
+++ b/drivers/base/power/opp/of.c
@@ -567,7 +567,6 @@ int dev_pm_opp_of_get_sharing_cpus(struct device *cpu_dev,
 				   struct cpumask *cpumask)
 {
 	struct device_node *np, *tmp_np, *cpu_np;
-	struct device *tcpu_dev;
 	int cpu, ret = 0;
 
 	/* Get OPP descriptor node */
